### PR TITLE
Bugfix: Remove inactive game modes from games per day chart in Statistics page

### PR DESCRIPTION
**Pull request by RickyMarou**
☝️ What the title says 

Before:

image

After:

image

We most likely have the same problem across the website in multiple places.
I would suggest filtering out these game modes from the response that the backend gives, or for a pure frontend fix we could have a util `getGameModes` somewhere but i would like to stay for from the state management for the foreseeable future 😇

### CODE REVIEW

File: src/components/overal-statistics/MultipleAmountPerDayChart.vue
Comment (gabcinder2004) [inline src/components/overal-statistics/MultipleAmountPerDayChart.vue:37]: Probably don't need these comments anymore?

**Review by gabcinder2004 — COMMENTED** [review]
Overall looks good to me. Consider dropping the console.logs though. :)

**Comment by modmoto** [comment]
is game modes undefined custom games? Would be cool in showing them aswell, right?

File: src/components/overal-statistics/MultipleAmountPerDayChart.vue
Comment (modmoto) [reply src/components/overal-statistics/MultipleAmountPerDayChart.vue:37]: +1

Comment (RickyMarou) [reply src/components/overal-statistics/MultipleAmountPerDayChart.vue]: Nice catch i let those slip through during my cleanup.

**Comment by RickyMarou** [comment]
> is game modes undefined custom games? Would be cool in showing them aswell, right?

I was wondering the same thing. Anybody can confirm that ? Then I would add it to the enum and show it.

Would be clearer if the backend was returning strings instead of ints for game modes 😬
Imagine if in the future there is footmen frenzy, dota, etc... it'll be hard to keep track of what's what.

**Comment by RickyMarou** [comment]
@modmoto what about merging like this and enabling when custom games stats are available ?
image

**Comment by modmoto** [comment]
Jeah, seems fine, thanks for the pr :)

**Comment by modmoto** [comment]
Hmm, don't find the merge button on mobile, will merge when I am back home.

**Review by modmoto — APPROVED** [review]
Ah here it is, after the review :D

**Pull request merged**
